pdm/__init__.py

```diff
@@ -17,6 +18,8 @@
     "Core",
 )
 
+os.system("")
```

Comment:
It would probably be good to add a link to the Python issue in a comment above that statement :slightly_smiling_face:

Reply:
will do